src/www/ui/css/select2.min.css

```diff
@@ -1 +1,7 @@
+/*
+ SPDX-FileCopyrightText: © Fossology contributors
+
+ SPDX-License-Identifier: GPL-2.0-only
```

Comment:
Please change license to MIT and add following copyrights:
`Copyright (c) 2012-2017 Kevin Brown, Igor Vaynberg, and Select2 contributors`

The file is licensed here: https://github.com/select2/select2/blob/master/LICENSE.md